fmriprep/workflows/bold/resampling.py

```diff
@@ -242,65 +216,25 @@ def select_target(subject_id, space):
     #
     # These depend on two optional steps: goodvoxel projection and medial wall nan replacement
     #
-    # inputnode -> optional(goodvoxels_bold_mask_wf) -> rename_src
-    # sampler -> optional(metric_dilate) -> optional(medial_nans) -> update_metadata
+    # inputnode -> rename_src
+    # sampler -> optional(medial_nans) -> update_metadata
     #
-    metric_dilate = pe.MapNode(
-        MetricDilate(distance=10, nearest=True),
-        iterfield=["in_file", "surf_file"],
-        name="metric_dilate",
-        mem_gb=mem_gb * 3,
-    )
-    metric_dilate.inputs.surf_file = [
-        str(
-            tf.get(
-                "fsaverage",
-                hemi=hemi,
-                density="164k",
-                suffix="midthickness",
-                extension=".surf.gii",
-            )
-        )
-        for hemi in "LR"
-    ]
 
     # Refine if medial vertices should be NaNs
     medial_nans = pe.MapNode(
         MedialNaNs(), iterfield=["in_file"], name="medial_nans", mem_gb=DEFAULT_MEMORY_MIN_GB
     )
 
-    if project_goodvoxels:
-        goodvoxels_bold_mask_wf = init_goodvoxels_bold_mask_wf(mem_gb)
-
-        # fmt: off
-        workflow.connect([
-            (inputnode, goodvoxels_bold_mask_wf, [
-                ("source_file", "inputnode.bold_file"),
-                ("anat_ribbon", "inputnode.anat_ribbon"),
-            ]),
-            (goodvoxels_bold_mask_wf, rename_src, [("outputnode.masked_bold", "in_file")]),
-            (goodvoxels_bold_mask_wf, outputnode, [
-                ("outputnode.goodvoxels_ribbon", "goodvoxels_ribbon"),
-            ]),
-            (sampler, metric_dilate, [("out_file", "in_file")]),
-        ])
-        # fmt: on
-    else:
-        workflow.connect([(inputnode, rename_src, [("source_file", "in_file")])])
+    workflow.connect([(inputnode, rename_src, [("source_file", "in_file")])])
```

Comment:
personal styling: seems wasteful to construct these objects for one connection

```suggestion
    workflow.connect(inputnode, "source_file", rename_src, "in_file")
```